src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -408,8 +426,8 @@ public void testRemoveImage() throws Exception {
     removedImages.addAll(sut.removeImage(imageVersion));
 
     assertThat(removedImages, hasItems(
-        new RemovedImage(UNTAGGED, imageLatest),
-        new RemovedImage(UNTAGGED, imageVersion)
+            new RemovedImage(UNTAGGED, imageLatest),
```

Comment:
Your IDE must be auto-reformatting some lines and in other places. Can we remove these formatting changes?